poglink/models/bans.py

```diff
@@ -0,0 +1,117 @@
+import datetime
+import re
+from dataclasses import dataclass
+from typing import List
+
+from dateutil import parser as dateparser
+from jinja2 import Template
+
+
+@dataclass
+class BansPlatformPair:
+    platform: str
+    nbans: int
+
+    def to_dict(self):
+        return {self.platform: self.nbans}
```

Comment:
As with `models/rates.py` I just split these out into their own separate modules as the other one was kinda unweildy to read through when working on only one of the two types of models.